### DISCUSSION TITLE
Turning off year grouping

### DISCUSSION THREAD

**Discussion by kylecordes** [jpanther/congo#264] [General]
From the intro, I am thrilled to see that Congo isn't blog-only, it is "great for both static page-based content".

But... the list template seems to be locked in to date-based ordering and optional year-based groups.

To get it around it I used a local `list.html`, but of course that is ugly.

Could Congo more fully support non-blogs, by making this (and anywhere else hardcoded to date-sort), have a non-blog non-date mode?

Hmmm..

**Comment by jpanther** [comment]
Thanks for the suggestion, @kylecordes. I'm open to implementing this sort of change so that the theme can be as flexible as possible. Is your `list.html` available in a repo somewhere so I can see what changes you made to suit your use case?
